packages/gatsby/src/utils/babel-loader-helpers.ts

```diff
@@ -16,13 +30,23 @@ const loadCachedConfig = () => {
   return pluginBabelConfig
 }
 
-const getCustomOptions = stage => {
+export const getCustomOptions = (stage: Stage): IBabelStage["options"] => {
   const pluginBabelConfig = loadCachedConfig()
   return pluginBabelConfig.stages[stage].options
 }
 
-const prepareOptions = (babel, options = {}, resolve = require.resolve) => {
-  let pluginBabelConfig = loadCachedConfig()
+type prepareOptionsType = (
```

Comment:
Rather than typing the whole function like this, can you just type the options object?